src/dba/models/Hashlist.class.php

```diff
@@ -122,6 +125,14 @@ function setSaltSeparator($saltSeparator) {
   function getCracked() {
     return $this->cracked;
   }
+
+  function getCreatorId() {
+    return $this->creator;
```

Comment:
Should be `$this->creatorId;`

Reply:
Fixed it here:
https://github.com/hashtopolis/server/pull/778/commits/4f004e63d7ee477285a67aa027c215ce3d03c8b6

My bad, forgot to stash the change, was already done locally.